index.html

```diff
@@ -5,11 +5,84 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <script src="https://kit.fontawesome.com/4a9487b343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./Components/components.css" />
+    <link rel="stylesheet" href="index.css">
+    <title>Booky UI</title>
 </head>
 
 <body>
 
+    <nav class="navbar">
+        <div class="nav-logo">
+            <img src="Assets/Logo/logo-64.png" alt="">
+        </div>
```

Comment:
img tag does not have closing / and alt tag is empty

Reply:
<img ... /> ? what is the purpose of the / ?